### PR TITLE
[DOC] shorthand methods and quotes for json-api serializer

### PR DESCRIPTION
**Pull request by knownasilya**

### CODE REVIEW

**Comment by locks** [comment]
Can't kick the AppVeyor build cc @igorT

File: addon/serializers/json-api.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const dasherize = Ember.String.dasherize;
   ```js
```
Comment (locks) [inline addon/serializers/json-api.js]: can you update this with the actual file path, like the rest?

File: addon/serializers/json-api.js [hunk 2]
```diff
@@ -24,30 +24,32 @@ const dasherize = Ember.String.dasherize;
 
   This serializer normalizes a JSON API payload that looks like:
 
-  ```js
+  ```app/models/player.js
+  import DS from 'ember-data';
 
-    // models/player.js
-    import DS from "ember-data";
+  const { attr, belongsTo } = DS;
```
Comment (locks) [inline addon/serializers/json-api.js]: can you revert the destructuring? ;X

Comment (knownasilya) [inline addon/serializers/json-api.js]: k :)

Comment (knownasilya) [inline addon/serializers/json-api.js]: Seems inconsistent, since they are used in other examples. Should those be removed?

Comment (locks) [inline addon/serializers/json-api.js]: Hm, I personally think so, but we should get some data team consensus.

Comment (locks) [inline addon/serializers/json-api.js]: (I thought the docs didn't use destructuring, like Ember/Guides)

**Pull request merged**